src/components/inline-editable/inline-editable.tsx

```diff
@@ -199,12 +203,12 @@ export class InlineEditable implements InteractiveComponent, LabelableComponent,
   //--------------------------------------------------------------------------
 
   /**
-   * Emitted when the cancel button gets clicked.
+   * Emits when the cancel editing button is pressed.
    */
   @Event({ cancelable: false }) calciteInlineEditableEditCancel: EventEmitter<void>;
 
   /**
-   * Emitted when the check button gets clicked.
+   * Emits when the confirm edits button is pressed.
```

Comment:
maybe `"confirm edits"`

Reply:
We were reserving the double quotes for property values:
> Use double quotes (") for the values of properties/attributes and event details.

WDYT about:

> Emits when the [component's] confirm edits button is pressed.

and

> Emits when the [component's] cancel editing button is pressed.

Reply:
hmm It seems to weird to reserve quotations for values. the tick in markdown is reserved for code values.

It seems like the button value needs to be surrounded by something. Could we use single quotes?

Reply:
This is good feedback, updated to double quotes.